pacehrh/tests/testthat/test-pace_lint.R

```diff
@@ -6,61 +6,68 @@ withr::local_dir("..")
 test_that("Lint: basic", {
   e <- pacehrh:::GPE
   local_vars("inputExcelFile", envir = e)
-  e$inputExcelFile <- "./config/model_inputs.xlsx"
-
+  e$inputExcelFile <- "../../config/model_inputs.xlsx"
   outFile = "lintout_01_clean.txt"
+
+  # Check that linting produces an output file
   errCode <- pacehrh::CheckInputExcelFileFormat(outputFile = outFile)
   testthat::expect_true(file.exists(outFile))
+
+  if (file.exists(outFile)){
```

Comment:
you could probably do this to make them remove automatically
```
with_file(outfile, {
   errCode <- pacehrh::CheckInputExcelFileFormat(outputFile = outFile)
   testthat::expect_true(file.exists(outFile))
})
```

Reply:
Great suggestion. Next time :-)